main.py

```diff
@@ -72,26 +62,24 @@ def write_azure_identity_binding(identityList):
         logger.debug(f'processing msi #{identity_id} containing')
 
         # Convert the data in the dictionary item into variables
-        filename = identity_info['msi'] + '.yaml'
+        filename = './output/' + identity_info['msi'] + '-binding.yaml'
         identityName = identity_info['msi']
-        resourceId = identity_info['Resource ID']
-        clientId = identity_info['Client ID']
 
         #Create a single YAML for the AzureIdentityBinding for each item in identityList
+        os.makedirs(os.path.dirname(filename), exist_ok=True)
         with open (filename, 'w') as identityfile:
             logger.debug(f'processing file:{filename}')
             identityfile.write(f'apiVersion: "aadpodidentity.k8s.io/v1"\n')
-            identityfile.write(f'kind: AzureIdentity\n')
+            identityfile.write(f'kind: AzureIdentityBinding\n')
             identityfile.write(f'metadata:\n')
-            identityfile.write(f'  name: {identityName}\n')
+            identityfile.write(f'  name: {identityName}-binding\n')
```

Comment:
next time, the tab can be achieved with a \t as well. Might be useful if you encounter a scenario where you need multiple tabs.